packages/expo-image/src/utils/resolveSources.tsx

```diff
@@ -3,7 +3,7 @@ import { resolveBlurhashString, resolveThumbhashString } from './resolveHashStri
 import { ImageNativeProps, ImageProps, ImageSource } from '../Image.types';
 
 export function isBlurhashString(str: string): boolean {
-  return /^(blurhash:\/)?[\w#$%*+,\-.:;=?@[\]^_{}|~]+(\/[\d.]+)*$/.test(str);
+  return /^(blurhash:\/)+[\w#$%*+,\-.:;=?@[\]^_{}|~]+(\/[\d.]+)*$/.test(str);
```

Comment:
I believe we should include that in the changelog too, as it appears to be a separate issue, correct?

Reply:
Yes, your right, I'll add it 👍

Reply:
This seems to break detecting if it's a blurhash when it doesn't start with `blurhash:/`. I think `?` was correct here.

Reply:
Maybe I didn't understand it was supposed to do then. We want it to take any string, and append `blur hash:/` to the start whether it starts with it or not? That breaks loading a local asset if that's the case. If I give a source of `image1`, I expect it to load a local asset named `image1`. Without this change it gets turned into `blurhash:/image1`.

Reply:
So we want to take any string and add blurhash:/? That regex will match everything at the moment.